tests/consequence_prediction/repeat_expansion_variants/test_pipeline.py

```diff
@@ -83,5 +83,7 @@ def test_alternative_identifiers():
 
 
 def test_missing_names():
-    """Records that are missing variant identifiers/names are skipped."""
-    assert run_pipeline('missing_names.xml.gz') == []
+    """Records that are missing variant names are still processed using HGVS identifier."""
+    results = run_pipeline('missing_names.xml.gz')
+    assert len(results) == 7
+    assert all(r[4] == 'short_tandem_repeat_expansion' for r in results)
```

Comment:
Is this testing the `explicity_insertion_lenght` behaviour?

Reply:
Good point, all of the nameless variants actually had HGVS in this case so it wasn't actually testing this path.  I'll add a test.